src/main/angular/app/graph/graph.component.ts

```diff
@@ -83,11 +87,26 @@ export class GraphComponent implements OnInit {
     await this.graphProcessor.processChanges(
       changes,
       this.properties,
-      this.dataService
+      this.showCumulativeDifference
     );
     this.shouldShowChart = this.projects.length > 0;
   }
 
+  toggleCumulativeDifference(change: MatSlideToggleChange) {
+    if (this.showCumulativeDifference) {
+      this.graphProcessor.removeCumulativeDifferences(
+        this.properties,
+        this.projects
+      );
+    } else {
+      this.graphProcessor.addCumulativeDifferences(
+        this.properties,
+        this.projects
+      );
+    }
+    this.showCumulativeDifference = !this.showCumulativeDifference;
```

Comment:
The method naming pattern seems a bit weird to me. 

the way that I'm reading this is: 
"if show cumulative difference" -> remove cumulative differences
else -> add cumulative differences. 

I would think that it would be "if show cumulative difference" -> add cumulative differences
else -> remove cumulative differences. 

Is there something I'm missing here?

Reply:
It's confusing - since I'm inverting the `showCumulativeDifference` var at the end, it's the opposite of what one would expect. I just changed it to behave as expected.